### DISCUSSION TITLE
复刻成功展示区

### DISCUSSION THREAD

**Discussion by txp666** [peng-zhihui/ElectronBot#141] [Show and tell]
空间实在太极限了，可以换细一点的线！最后大力出奇迹还是有一条缝
还有摄像头发热严重

https://user-images.githubusercontent.com/52451470/162608656-ae786ba9-09d9-4520-8760-ab658e50aa64.mp4

**Comment by AIxer** [comment]
大佬能以博客或者视频的方式发个复刻教程么？

**Reply by txp666** [reply]
https://www.bilibili.com/video/BV1g94y1d7BL?share_source=copy_web
也不敢说教程！就是记录一下！有些问题是公共的有些问题有的人会遇到，issue里面基本上比较全，但是可能有的同学不会一条一条仔细看，就大概说个流程！而且剪辑也不太好。。。

**Reply by AIxer** [reply]
谢谢！

**Comment by wenFrankieeee** [comment]
https://user-images.githubusercontent.com/101490638/165811148-013e8031-3368-4f69-8db8-be39e7d39c11.mp4

我完成了PZH 两个项目得复刻

**Reply by carveybunt** [reply]
大佬，弱弱的问下，复刻2个项目，材料费用多少，能否一站式采购？我也想复刻

**Comment by isSJ** [comment]
实在是塞不进去。线和胶打多了。发热严重。测试很好，一组装就断了，没办法，退而求其次改装成了减配两个侧向舵机。乞丐版很顺利安装完毕。节约50元。相应固件也注释掉了减掉舵机的控制和反馈部分。不然外围现有应用无法连接。

**Reply by isSJ** [reply]
手机发送视频失败。抽空补上。看不出来有太大区别

**Comment by txp666** [comment]
https://user-images.githubusercontent.com/52451470/168427827-69fea2e0-aa13-4e15-9e0c-d51cdb921c9e.mp4

又复刻了一个plus版

**Comment by GreenShadeZhang** [comment]
https://user-images.githubusercontent.com/24240675/169656540-571f22df-eef3-4395-bb94-f8aab82a47ec.MP4
